### PR TITLE
Port privacy archive pages to Protocol (Fixes #7515)

### LINKED ISSUE #7515: Port archived privacy pages to Protocol.
## Description

The archived privacy policy pages are still using Sandstone:

https://www.mozilla.org/en-US/privacy/archive/

Whilst these notices are archived and their content should not be updated, I don't think that should prevent us from removing the legacy framework those pages are using.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Pages are updated in production using Protocol.

#### Tasks
- [ ] Update templates.
- [ ] Remove legacy assets.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Ports [privacy archive](https://www.mozilla.org/en-US/privacy/archive/) pages from Sandstone to Protocol

## Issue / Bugzilla link
#7515

## Testing
URLs:

- http://localhost:8000/en-US/privacy/archive/
- http://localhost:8000/en-US/privacy/archive/firefox/2006-10/
- http://localhost:8000/en-US/privacy/archive/firefox/2008-06/
- http://localhost:8000/en-US/privacy/archive/firefox/2009-01/
- http://localhost:8000/en-US/privacy/archive/firefox/2009-09/
- http://localhost:8000/en-US/privacy/archive/firefox/2010-01/
- http://localhost:8000/en-US/privacy/archive/firefox/2010-12/
- http://localhost:8000/en-US/privacy/archive/firefox/2012-12/
- http://localhost:8000/en-US/privacy/archive/firefox/2013-05/
- http://localhost:8000/en-US/privacy/archive/firefox/third-party/
- http://localhost:8000/en-US/privacy/archive/firefox-cliqz/2018-06/
- http://localhost:8000/de/privacy/archive/firefox-cliqz/2018-06/
- http://localhost:8000/en-US/privacy/archive/firefox-marketplace/2015-01/
- http://localhost:8000/en-US/privacy/archive/hello/2014-11/
- http://localhost:8000/en-US/privacy/archive/hello/2016-03/
- http://localhost:8000/en-US/privacy/archive/persona/2017-07/
- http://localhost:8000/en-US/privacy/archive/thunderbird/2010-06/
- http://localhost:8000/en-US/privacy/archive/websites/2013-08/

### CODE REVIEW

File: bedrock/privacy/templates/privacy/archive/websites-2013-08.html [hunk 1]
```diff
@@ -8,98 +8,97 @@
...
-        <h2>Privacy Policy Changes</h2>
-        <p>By using the websites, you agree to the terms and conditions of this privacy policy. If you do not agree to the terms and conditions of this privacy policy, please do not use the websites. Mozilla may change its Privacy Policy from time to time. Any and all changes will be reflected on this page. Substantive changes will also be announced through the standard mechanisms through which Mozilla communicates with the Mozilla community, including Mozilla's “announce” <a href="https://lists.mozilla.org/listinfo/announce">mailing list</a> and <a href="https://groups.google.com/group/mozilla.announce">newsgroup</a>. </p>
-      </section>
-      <section>
-        <h2>For More Information</h2>
-        <p>If you have questions about this privacy policy, please contact the Mozilla Foundation at privacy@mozilla.org or the Mozilla Corporation at privacy@mozilla.com.</p>
-      </section>
-    </div>
-  </article>
+{% block title %}Websites Privacy Policy{% endblock %}
```
Comment (stephaniehobson) [inline bedrock/privacy/templates/privacy/archive/websites-2013-08.html]: This should have the Mozilla theme to match the current Websites Privacy Policy.

File: bedrock/privacy/templates/privacy/archive/index.html [hunk 2]
```diff
@@ -8,66 +8,63 @@
...
-      </ul>
-    </section>
-    <section>
-      <h2>{{ _('Mozilla Websites') }}</h2>
-      <ul>
-        <li><a href="{{ url('privacy.archive.websites-2013-08') }}"><time datetime="2013-08">{{ _('August 2013') }}</time></a></li>
-      </ul>
-    </section>
-  </article>
+{% block title %}{{ _('Outdated Policies') }}{% endblock %}
```
Comment (stephaniehobson) [inline bedrock/privacy/templates/privacy/archive/index.html]: The index page feels like it should be Mozilla themed.

**Comment by alexgibson** [comment]
@stephaniehobson thanks for the review and suggestions! I pushed a followup commit with some fixes. I think I'm going to draw a line at adding in logos that weren't there when the policies were archived, but I think I should have addressed all other comments.

File: bedrock/privacy/templates/privacy/archive/base-resp.html [hunk 3]
```diff
@@ -3,48 +3,23 @@
...
+    <h2 class="side-reference-title">
+      <a href="{{ url('privacy.archive.index') }}">{{ _('Outdated Policies') }}</a>
+    </h2>
+  </section>
 {% endblock %}
 
-{% block email_form %}{% endblock %}
-
-{% block js %}
-  {{ js_bundle('privacy') }}
```
Comment (alexgibson) [inline bedrock/privacy/templates/privacy/archive/base-resp.html]: Note to self: I may be able to kill this bundle

**Pull request merged**